src/process/init.rs

```diff
@@ -264,6 +278,11 @@ pub fn container_init(args: ContainerInitArgs) -> Result<()> {
         cleanup_file_descriptors(preserve_fds).with_context(|| "Failed to clean up extra fds")?;
     }
 
+    // change directory to process.cwd if process.cwd is not empty
+    if do_chdir {
```

Comment:
I felt like it would always be false here, but in what cases would it be executed?

Reply:
Oh, sorry, I forgot to delete line 219.
I'll fix it right away.
This chdir will be executed when the first chdir fails by EPERM.